deps/build.jl

```diff
@@ -4,9 +4,12 @@ using MPI
 import Pkg.TOML
 import P4est_jll
 
+# Only required on MacOS systems
+const xcode_include_path_cli = "/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include/"
+const xcode_include_path_gui = "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/"
```

Comment:
If you need these only in the `if` block below, I would prefer moving these definitions there.

Reply:
Fixed in aa98636.